### PR TITLE
Model config keeps NUM_REALIZATIONS

### PR DESCRIPTION
**Pull request by markusdregi**
**Task**
_Yet another PR on the path towards no config loading within enkf_main. Let model_config keep the number of realizations._

**Pre un-WIP checklist**
- [x] Statoil tests pass locally

### CODE REVIEW

File: libenkf/src/enkf_main.c [hunk 1]
```diff
@@ -2166,6 +2166,11 @@ static void enkf_main_init_obs(
   }
 }
 
+static void enkf_main_init_ensemble(enkf_main_type * enkf_main) {
```
Comment (joakim-hove) [inline libenkf/src/enkf_main.c]: This is a *very limited* initialization - I fear the name creates confusion. What about: `enkf_main_add_ensemble_members( )`?

File: python/python/res/enkf/model_config.py [hunk 2]
```diff
@@ -118,3 +119,7 @@ def getEnspath(self):
     def getRunpathFormat(self):
         """ @rtype: PathFormat """
         return self._get_runpath_fmt()
+
+    @property
```
Comment (joakim-hove) [inline python/python/res/enkf/model_config.py:123]: Discussion: are we all in on properties? I thought no!

**Review by joakim-hove — COMMENTED** [review]
This can be merged - but:

1. The `enkf_main_init_ensemble( )` function should be renamed.

2. We should have a common agreement on the use properties.

File: libenkf/src/enkf_main.c (hunk 1 above)
Comment (markusdregi) [reply libenkf/src/enkf_main.c]: Sure, I'll change it

File: python/python/res/enkf/model_config.py (hunk 2 above)
Comment (markusdregi) [reply python/python/res/enkf/model_config.py:123]: I thought yes. So lets have a discussion on this after stand up..

**Review by joakim-hove — APPROVED** [review]

**Pull request merged**